.drone.yml

```diff
@@ -172,6 +172,27 @@ steps:
           - feature/*
       event: [push, pull_request]
 
+# Trivy Security Scannner
+
+  - name: scan-image
+    pull: always
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
+    resources:
+      limits:
+        cpu: 1000
+        memory: 1024Mi
+    environment:
+      IMAGE_NAME: ima:${DRONE_COMMIT_SHA}
+      TOLERATE: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
```

Comment:
I don't think we should tolerate high and critical  and should we add a WHITELIST_FILE for privy exceptions?

Reply:
@Rhodine-orleans-lindsay We don't have an option to use Tolerate which was my mistake. I have replaced it now with SEVERITY. We can block the pipeline from progressing to next steps after seeing High and Critical Vulnerabilties.  But this will result in all the Pipelines in broken state, unless those vulnerabilities are fixed which could take 2-3 days to fix and test them. There are high number of vulnerabilities When we investigated for libwebp. May be once we update all the Images we can use the option. However, There are seperate tickets in another epic which covers the fixing of vulnerabilties.